src/AutoRest.CSharp/Generation/Writers/LowLevelClientWriter.cs

```diff
@@ -115,7 +114,8 @@ private void WriteClientMethod(CodeWriter writer, RestClientMethod clientMethod,
 
         private void WriteClientFields(CodeWriter writer, LowLevelRestClient client, BuildContext context)
         {
-            writer.Line($"private readonly {typeof(HttpPipeline)} {PipelineField};");
+            writer.Append($"protected {typeof(HttpPipeline)} {PipelineField}");
+            writer.AppendRaw("{ get; }\n");
```

Comment:
Can we add a test project to see these changes and add some tests for it?

Reply:
I'm trying to wait for LLC output to be stabilized before adding a test project. 

Maybe I should give in and add one, thoughts @pakrym ?

Reply:
https://github.com/Azure/autorest.csharp/pull/1109#issuecomment-812012866 😄

Reply:
I didn't mean to add a test project in `autorest.testserver` repo. I meant we can add test project [here](https://github.com/Azure/autorest.csharp/tree/feature/v3/test/TestProjects), if you see here we have only c# related test projects. We can add/update projects anytime here as per our needs.

The plus side of adding a project and tests would to see the differences made by the PR and see if the changes broke any tests.

Reply:
I added a top level sample for this PR.

As discussed in the call, detailed swaggers are coming later. I've file https://github.com/Azure/autorest.csharp/issues/1115 to make sure that work doesn't get lost.